### ISSUE TITLE
Unable to run npm 6.4.1 in integrated terminal

### ISSUE THREAD

**Issue by YoshiJL** [microsoft/vscode#179407]
Type: <b>Bug</b>

Run `npm run dev`.

**Expected:** `npm run dev` runs and Node app starts.

The following error is outputed:
```
SyntaxError: Unexpected token .
    at new Script (vm.js:79:7)
    at createScript (vm.js:251:10)
    at Object.runInThisContext (vm.js:303:10)
    at Module._compile (internal/modules/cjs/loader.js:657:28)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at Module._preloadModules (internal/modules/cjs/loader.js:812:12)
```

VS Code version: Code 1.77.1 (b7886d7461186a5eac768481578c1d7ca80e2d21, 2023-04-04T23:21:11.906Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz (16 x 2304)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|31.73GB (16.04GB free)|
|Process Argv|--crash-reporter-id 3a98b62f-57cd-42de-9943-7c4003a4cc81|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (43)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-sql-formatter|adp|1.4.4
vscode-icalendar|af4|1.0.1
TabOut|alb|0.2.2
Bookmarks|ale|13.3.1
pug|ama|1.0.1
html-entities|chr|1.1.2
cypress-snippets|Cli|0.0.5
csharpier-vscode|csh|1.3.6
vue-peek|dar|1.0.2
vscode-eslint|dba|2.4.0
gitlens|eam|13.4.0
vsc-material-theme|Equ|33.8.0
prettier-vscode|esb|9.10.4
todo-tree|Gru|0.0.224
vue-snippets|hol|1.0.4
npm|ide|1.3.0
vue|jcb|0.1.5
vscode-peacock|joh|4.2.2
cypress-fixture-intellisense|Jos|1.2.0
vscode-liquid-snippets|kil|2.0.3
regexp-preview|Lou|0.1.5
template-string-converter|meg|0.6.0
fluent-icons|mig|0.0.18
vscode-filesize|mkx|3.1.0
live-server|ms-|0.4.7
powershell|ms-|2023.3.3
vsliveshare|ms-|1.0.5834
vsliveshare-pack|ms-|0.4.0
vscode-react-native|msj|1.10.2
copy-json-path|nid|0.3.0
fix-json|oli|0.1.2
material-icon-theme|PKi|4.25.0
vscode-sort-json|ric|1.20.0
vue-vscode-snippets|sdr|3.1.1
vscode-cy-helper|She|1.2.2
vsc-workspace-sidebar|ske|1.4.2
code-spell-checker|str|2.20.4
language-stylus|sys|1.16.0
lorem-ipsum|Tyr|1.3.1
terascript-syntax-highlight|und|1.0.2
markdown-all-in-one|yzh|3.5.1
json|Zai|2.0.2
html-css-class-completion|Zig|1.20.0

(3 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30256860
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
vsccc:30610678
nodejswelcome1cf:30587006
2e4cg342:30602488
pyind779:30671433
89544117:30613380
pythonsymbol12:30671437
vsccsb:30705552
defaultazurewalk:30687958
pythonms35:30701012
pythonclientmv:30705673

```

</details>

**Comment by meganrogge** [comment]
I believe this is already fixed, can you see if you can reproduce in the [Insiders build](https://code.visualstudio.com/insiders)?
/label info-needed

**Comment by YoshiJL** [comment]
Thanks! This is indeed fixed in 1.78.0-insider! I'll use Terminal for now while I wait for the next release.
